test/conftest.py

```diff
@@ -238,7 +239,9 @@ def __init__(self, blob_host, blobs):
 
         @self.function_body
         def default_function_body(*args, **kwargs):
-            return sum(arg**2 for arg in args) + sum(value**2 for key, value in kwargs.items())
+            return sum(arg**2 for arg in args if isinstance(arg, numbers.Number)) + sum(
```

Comment:
instead of making this default more complex could you instead set `function_body` below just for the grouped function?